en/docs/administer/managing-users-and-roles/managing-user-stores/configuring-secondary-user-stores.md

```diff
@@ -29,19 +29,19 @@ There are two approaches to configure a secondary user store. It can be configur
     </thead>
     <tbody>
     <tr class="odd">
-    <td><code>org.wso2.carbon.user.core.ldap.ReadOnlyLDAPUserStoreManager</code></td>
+    <td><code>org.wso2.carbon.user.core.ldap.UniqueIDReadOnlyLDAPUserStoreManager</code></td>
```

Comment:
Since this was ldap we might have to keep both class names in here
@prasa7 wdyt ?.

Reply:
So in the table we need to include both old user store manager (ReadOnlyLDAPUserStoreManager) and user store manager(UniqueIDReadOnlyLDAPUserStoreManager) for all the user store managers?

Reply:
I have added both old and new user store manager names